packages/dataviews/src/style.scss

```diff
@@ -185,6 +185,10 @@
 			min-height: $grid-unit-40;
 			display: flex;
 			align-items: center;
+
+			> * {
```

Comment:
@jameskoster I gave it a try to be able to render block previews with this, but I'm not 100% sure if it's the right solution. 

In general the previews issue stems from [useResizeObserver](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/block-preview/auto.js#L129), where it seems it can determine a width if we have just the `display:flex`. I'm not sure if @ciampo has any thoughts on this about whether it's something that should be handled by the hook.

Reply:
Thanks Nik. Visually this looks a lot better. Subject to Marco's feedback we can probably add a fixed height to the previews in table layout so they are consistent. It would also be nice to apply the theme background color to the container to avoid these awkward gray spaces in grid layout:

![grey](https://github.com/WordPress/gutenberg/assets/846565/d957d0ac-aa17-49e7-aa1c-4e4e1ccd62b8)